### PR TITLE
Use standard stress utility in examples

### PR DESCRIPTION
**Pull request by glikson**
In memory allocation examples, replaced the use of custom stress utility with the standard one (packaged in a small Alpine-based image).

### CODE REVIEW

File: docs/tasks/configure-pod-container/memory-request-limit-2.yaml [hunk 1]
```diff
@@ -6,16 +6,11 @@ metadata:
...
         memory: "100Mi"
-    args:
-    - -mem-total
-    - 250Mi
-    - -mem-alloc-size
-    - 10Mi
-    - -mem-alloc-sleep
-    - 1s
+    command: ["stress"]
+    args: ["--vm", "1", "--vm-bytes", "250M", "--vm-hang", "1"]
```
Comment (heckj) [inline docs/tasks/configure-pod-container/memory-request-limit-2.yaml]: is `--vm-hang` a valid variable? When I looked at https://hub.docker.com/r/polinux/stress/, it wasn't listed as an option. Have you run this example to verify it works as expected?

Comment (glikson) [reply docs/tasks/configure-pod-container/memory-request-limit-2.yaml]: I did. It is documented at https://linux.die.net/man/1/stress

**Pull request merged**